backend/app/Enums/Role.php

```diff
@@ -10,4 +10,9 @@ enum Role: string
     case Moderator = "moderator";
     case Company = "company";
     case Banned = "banned";
+
+    public function label(): string
+    {
+        return __($this->value);
+    }
 }
```

Comment:
Look at this package - https://spatie.be/docs/laravel-permission/v5/introduction

Reply:
Alright. That might be useful. I found maintained [vue package](https://www.npmjs.com/package/laravel-permission-to-vuejs) that will be needed for .vue files.

I'll make issue with that after PR.